src/containers/Auth/ConfirmOTP/ConfirmOTP.tsx

```diff
@@ -8,13 +8,20 @@ import { Auth } from '../Auth';
 import { Input } from '../../../components/UI/Form/Input/Input';
 import { sendOTP } from '../../../services/AuthService';
 
+// let's define registration success message
+const successMessage = (
+  <div>
+    Your account is registration successfully. Please contact your organisation admin for the
```

Comment:
I guess the typo error here, the sentence would be like - Your account is registered successfully....

Reply:
Fixed :)